.github/actions/zenodo-upload/zenodo_upload.py

```
@@ -0,0 +1,101 @@
+                f"{base_url}/{new_deposition_id}/files",
+                headers={"Authorization": f"Bearer {token}"},
+                files={"file": f}
+            )
+        if response.status_code != 201:
+            print(f"Error uploading {file_path}: {response.text}")
+            sys.exit(1)
+
+    # Publish
+    response = requests.post(f"{base_url}/{new_deposition_id}/actions/publish", headers=headers)
+    if response.status_code != 202:
+        print(f"Error publishing: {response.text}")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)